src/ui/core/zcl_abapgit_html.clas.abap

```diff
@@ -381,7 +390,12 @@ CLASS zcl_abapgit_html IMPLEMENTATION.
         TRY.
             lo_html ?= ig_chunk.
           CATCH cx_sy_move_cast_error.
-            ASSERT 1 = 0. " Dev mistake
+            TRY.
+                li_renderable ?= ig_chunk.
+                lo_html ?= li_renderable->render( ).
```

Comment:
how about this instead of passing on the exception:

```abap
try.
    li_renderable ?= ig_chunk.
    lo_html ?= li_renderable->render( ).
  catch zcx_abapgit_exception into lx_error.
    lo_html = zcl_abapgit_html=>create( |<span class="error">Render error: { lx_error->get_text( ) }</span>| ).
endtry.
```

Reply:
Interesting idea.

Reply:
done + UT